images/dockerregistry/Dockerfile

```diff
@@ -6,16 +6,20 @@
 #
 FROM openshift/origin-base
 
-COPY config.yml $REGISTRY_CONFIGURATION_PATH
-COPY bin/dockerregistry /dockerregistry
+RUN INSTALL_PKGS="origin-dockerregistry" && \
+    yum install -y ${INSTALL_PKGS} && \
+    rpm -V ${INSTALL_PKGS} && \
+    yum clean all
 
-LABEL io.k8s.display-name="OpenShift Origin Image Registry" \
-      io.k8s.description="This is a component of OpenShift Origin and exposes a Docker registry that is integrated with the cluster for authentication and management."
+COPY config.yml ${REGISTRY_CONFIGURATION_PATH}
+
+LABEL io.openshift.tags="openshift,docker,registry" \
+      io.k8s.display-name="OpenShift Container Platform Image Registry" \
+      io.k8s.description="This is a component ofOpenShift Container Platform and exposes a Docker registry that is integrated with the cluster for authentication and management."
```

Comment:
Typo, also, these are origin images, so "Container Platform" isn't really appropriate. 

We should use "OpenShift" everywhere in these.